Integrations/integration-FidelisElevateNetwork.yml

```diff
@@ -0,0 +1,777 @@
+commonfields:
+  id: Fidelis Elevate Network
+  version: -1
+name: Fidelis Elevate Network
```

Comment:
- is there an option to get the hash in malware alert
- separate upload-file to be upload-file and upload-url
-